src/app/app.component.css

```diff
@@ -0,0 +1,292 @@
+  :host {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    font-size: 14px;
+    color: #333;
+    box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
```

Comment:
This looks ok but I'm sending this back as an example.

Reply:
I'll look into making the requested changes.